packages/core/src/shared/constants.ts

```diff
@@ -189,6 +189,14 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
 
+// URLs for Amazon EventBridge Scheduler
+export const eventBridgeSchedulerCreateScheduleUrl: string =
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html'
+export const eventBridgeSchedulerApiGatewayUrl: string = 
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/managing-schedule.html'
+export const eventBridgeSchedulerRolePermissionsUrl: string =
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/setting-up.html'
+
 /**
  * Names of directories relevant to the crash reporting functionality.
  *
```

Comment:
The addition of EventBridge Scheduler URLs to the constants file is a good practice. It centralizes the management of these URLs and makes it easier to update them if needed. However, consider adding a comment above this section to clearly indicate that these are EventBridge Scheduler related constants, improving code readability.
```suggestion
export const amazonQHelpUrl = ''
export const amazonQVscodeMarketplace =
    ''

/**
 * URLs for Amazon EventBridge Scheduler documentation
 */
export const eventBridgeSchedulerCreateScheduleUrl: string =
    ''
export const eventBridgeSchedulerApiGatewayUrl: string = 
    ''
export const eventBridgeSchedulerRolePermissionsUrl: string =
    ''

/**
 * Names of directories relevant to the crash reporting functionality.
 */
```